src/Clock.tsx

```diff
@@ -0,0 +1,66 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timer = 0;
+
+  componentDidMount() {
+    this.timer = window.setInterval(() => {
+      this.setState({ date: new Date() });
+    }, 1000);
```

Comment:
you can console time here
```suggestion
    this.timer = window.setInterval(() => {
      const newDate = new Date();
      this.setState({ date: newDate });
      
      console.info(newDate.toLocaleTimeString());
    }, 1000);
```